.circleci/settings.py

```diff
@@ -0,0 +1,117 @@
+"""
+This script imports all of the required environment variables. If you are
+running it locally, update the .env file with the environment variables only
+available on your ci playform and run source.
+
+Usage: source .env && python3.6 initialize.py
+"""
+
+import os
+
+# PREFIX is the prefix of the name of the review apps created and deleted
+# by these scripts. PREFIX is used by `delete.py` to determine which apps
+# to delete after inactivity. So, it's important to use a prefix that won't
+# be used by Dash developers when creating apps manually, otherwise this
+# script may delete those apps!
+PREFIX = "review-app-"
+
+# LAST_UPDATE is the allowed amount of time before review apps are
+# deleted from the server.
+LAST_UPDATE = {"minutes": 1}
+
+# TRUNK_BRANCHNAME is the name of your repository's "main" or "master"
+# branch.
+TRUNK_BRANCHNAME = "main"
+
+# BRANCHNAME is the name of the branch you will initialize your Dash apps
+# from.
+BRANCHNAME = os.getenv("CIRCLE_BRANCH")
+
+# REPONAME is the name of the repository that will hold the review app
+# branches.
+REPONAME = os.getenv("CIRCLE_PROJECT_REPONAME")
+
+# TARGET_APPNAME is the name the Dash App that will serve as a review app
+# template. This script will copy that apps configuration settings and
+# apply them to all review apps.
+TARGET_APPNAME = "qa-1"
+
+# APPNAME determines how your review apps will be named.
+APPNAME = f"{TARGET_APPNAME}-rev-{BRANCHNAME}"[0:30]
```

Comment:
If `TARGET_APPNAME` is 24-30 characters, then every review app will be a repeat (as `BRANCHNAME` won't be applied to this string).

We could solve by having it be something like:
```
PREFIX = f"{TARGET_APPNAME[:15]}-rev-"
APPNAME = f"{PREFIX}{BRANCHNAME}"[:30]
```

That way, there will always be at least 10 characters used from `BRANCHNAME`.

Then, you'll need to use `PREFIX` within the `delete.py` code.